src/dappsExplorer/DAppsExplorerScreenSearchFilter.tsx

```diff
@@ -69,45 +70,48 @@ export function DAppsExplorerScreenSearchFilter() {
   const language = useSelector(currentLanguageSelector)
   const nonFavoriteDappsWithCategoryNames = useSelector(nonFavoriteDappsWithCategoryNamesSelector)
   const favoriteDappsWithCategoryNames = useSelector(favoriteDappsWithCategoryNamesSelector)
-  const [selectedFilter, setSelectedFilter] = useState('all')
 
-  // Some state lifted up from all and favorite sections
-  const [searchTerm, setSearchTerm] = useState('')
-  const [favoriteResultsEmpty, setFavoriteResultsEmpty] = useState(false)
-  const [allResultEmpty, setAllResultEmpty] = useState(false)
+  const [filterChips, setFilterChips] = useState<FilterChip<DappWithCategoryNames>[]>(
+    categories.map((category) => ({
+      id: category.id,
+      name: category.name,
+      filterFn: (dapp: DappWithCategoryNames) => dapp.categories.includes(category.id),
+      isSelected: false,
+    }))
+  )
```

Comment:
nit: this way it only does the mapping once when it needs the initial value
```suggestion
  const [filterChips, setFilterChips] = useState<FilterChip<DappWithCategoryNames>[]>(
    () => 
      categories.map((category) => ({
        id: category.id,
        name: category.name,
        filterFn: (dapp: DappWithCategoryNames) => dapp.categories.includes(category.id),
        isSelected: false,
      }))
  )
```

Reply:
oops, thanks for the catch!